doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,75 +3,70 @@
 Chapel Quickstart Instructions
 ==============================
 
-These instructions are designed to help you get started with
-a source distribution of the latest Chapel release.
+These instructions are designed to help users get started with a
+source distribution of Chapel as quickly as possible.  If you are
+interested in taking the time to understand Chapel's configuration
+options, build process, and installation more thoroughly, please refer
+to :ref:`readme-chplenv` and :ref:`readme-building` instead.
 
-In the following instructions, note that building and using Chapel as
-described in steps 2-5 disables optional and advanced features in the
-interest of getting you a clean build as quickly as possible. Later
-sections explain how to re-build in the preferred configuration and how to
-enable more features, such as distributed memory execution.
+These instructions first have you build a minimal, stripped-down
+version of Chapel to reduce build times and the potential for
+confusion.  Once you are interested in a full-featured version of
```

Comment:
I actually am interested in reducing user confusion (and think things like choosing whether to opt into or out of LLVM have the potential to cause confusion), but I agree that it may come across as judgmental / a chip to knock off of someone's shoulder.  For brevity how about just "to reduce build times and the potential for portability issues." ?

Reply:
How about take off "and the potential for portability issues."?  I prefer we not suggest to people that Chapel might not be portable, when the issue really is concern about other packages.

Reply:
That's a good point.  Maybe "third-party portability issues" to try and get the best of both worlds?

Reply:
That would work.